index.html

```diff
@@ -15,26 +15,201 @@
 </head>
 
 <body>
-    <nav>
-      <i class='bx bx-menu bx-sm'></i>
-    </nav>
+  <nav>
+    <i class='bx bx-menu bx-sm'></i>
+  </nav>
   <section class="headline">
     <div class="container">
       <h1>Hey there. <br>
         I’m Tsige.</h1>
-      <h2 class="green">I’m a software developer! </h2></div>
-      <p>I can help you build any software as your need
-        including DesktopApp ,Mobile feature or website.
-        For more Please Look through
-        some of my work and experience that I have!</p><br>
-      <ul>
-        <li><a href="https://twitter.com/TsigeWo"><img src="icons/twitter.svg"></a></li>
-        <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/"><img src="icons/Linkedin icon.svg"></i></a></li>
-        <li><a href=""><img src="icons/medium.svg"></i></a></li>
-        <li><a href="https://github.com/tsigewossne"><img src="icons/github.svg"></a></li>
-        <li><a href=""><img src="icons/angellist.svg"></a></li>
-      </ul>
+      <h2 class="green">I’m a software developer! </h2>
     </div>
-      </section>
+    <p>I can help you build any software as your need
+      including DesktopApp ,Mobile feature or website.
+      For more Please Look through
+      some of my work and experience that I have!</p><br>
+    <div><ul>
+      <li><a href="https://twitter.com/TsigeWo"><img src="icons/twitter.svg"></a></li>
+      <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/"><img src="icons/Linkedin icon.svg"></i></a></li>
+      <li><a href=""><img src="icons/medium.svg"></i></a></li>
+      <li><a href="https://github.com/tsigewossne"><img src="icons/github.svg"></a></li>
+      <li><a href=""><img src="icons/angellist.svg"></a></li>
+    </ul></div>
+  </section>
+  <section class="next"><img src="icons/Disabled.svg" alt="next"></section>
+    <section class="works">
+      <h2>My Recent works</h2>
+      <h2>
+        <hr>
+      </h2>
+      <div class="container1">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please fix your languages badges and stick to the Figma design as much as possible so kindly fix your design :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face:

**Your Design**
![Screenshot from 2023-06-03 14-00-50](https://github.com/tsigewossne/portfolio/assets/45226848/23eaed74-6416-40d4-ae70-b330d7e5cf59)

**Figma Design** 
![Screenshot from 2023-06-03 14-01-26](https://github.com/tsigewossne/portfolio/assets/45226848/3f867671-3bad-467b-9adc-e3dfecc16126)

- [ ] Also there is lot of space on the right side so kindly fix this spacing issue :+1:

![Screenshot from 2023-06-03 14-00-27](https://github.com/tsigewossne/portfolio/assets/45226848/bc4cfdfb-3b12-43d5-a231-06173dd4516d)

Reply:
Dear @aamirkhan2478  what is wrong with your browser
![desktop](https://github.com/tsigewossne/portfolio/assets/100133784/675aad2b-6cdf-4f78-99ba-ba2bf72309fe)
![Ruby](https://github.com/tsigewossne/portfolio/assets/100133784/fc5ba079-9c6a-4779-8ed7-0125a07c2beb)
? I have changed multiple browsers and checked if there is white space to the left but I can't get please review again this is my last chance for extra review but i can't get the issue you have said. I have attached the screenshot